UsageStatistics_Plugin_Admin.php

```diff
@@ -105,15 +105,15 @@ public function w3tc_ajax_ustats_get() {
 	 * Ajax: Test access log path.
 	 */
 	public function w3tc_ajax_ustats_access_log_test() {
-		$nonce = isset( $_REQUEST['_wpnonce'][0] ) ? sanitize_key( $_REQUEST['_wpnonce'][0] ) : false;
+		$nonce = isset( Util_Request::get_array( '_wpnonce')[0] ) ? Util_Request::get_array( '_wpnonce' )[0] ) : false;
 
 		if ( ! wp_verify_nonce( $nonce, 'w3tc' ) ) {
 			wp_die( esc_html__( 'Invalid WordPress nonce.  Please reload the page and try again.', 'w3-total-cache' ) );
 		}
 
 		$handle   = false;
-		$filepath = isset( $_REQUEST['filename'] ) ?
-			str_replace( '://', '/', sanitize_text_field( wp_unslash( $_REQUEST['filename'] ) ) ) : null;
+		$filepath = isset( Util_Request::get_string( 'filename' ) ) ?
```

Comment:
Use `! empty()`.  You may want to set a variable to avoid calling twice.

Reply:
Changed to ! empty and created a variable to avoid 2 function calls.